web-app/pages/certificate/[tokenId].tsx

```diff
@@ -38,16 +39,31 @@ function buildLinkedInUrl(pngUrl: string) {
 //   return { props };
 // };
 
-const Certificate: NextPage = () => {
+// eslint-disable-next-line max-lines-per-function
+const Certificate: NextPage = (...props) => {
   const router = useRouter();
-  const { tokenId } = router.query; // https://nextjs.org/docs/routing/dynamic-routes
-
-  console.log({ tokenId });
-  const pngUrl = `${baseUrl}/api/cert/${tokenId}.png`;
-
+  const { tokenId } = router.query; // https://nextjs.org/docs/routing/dynamic-routes  
+  const data = tokenId;
+  const pngUrl = `${baseUrl}/api/cert/${data}.png`;
+  // eslint-disable-next-line react/destructuring-assignment
+  console.log('************************', { data });
```

Comment:
@rashaabdulrazzak  I don't understand these lines. Why not just use `tokenId` instead of defining a copy of it as `data`?

Reply:
yeah the problem is when I use the` tokenId` directly it gives me `undefined` as a value that's why I assigned it to another variable